packages/contracts/hardhat.config.ts

```diff
@@ -108,6 +110,10 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: process.env.ETHERSCAN_API_KEY,
   },
+  dodoc: {
+    runOnCompile: true,
+    testMode: false,
```

Comment:
Since test mode is correctly disabled here we shouldn't have the docs jsons checked in. Those are only generated for testing purposes. Please remove